src/prefect/orion/schemas/core.py

```diff
@@ -559,6 +559,16 @@ def validate_name_characters(cls, v):
         raise_on_invalid_name(v)
         return v
 
+    @validator("is_schedule_active", pre=True)
+    def validate_is_schedule_active(cls, v, field):
+        """
+        Primary responsibility of this validator is to make sure that actions that use this field
+        are able to safely accept a `None` value.
+        """
+        if v is None:
+            return field.default
+        return v
```

Comment:
I'm pretty confused by this validator, it seems like a no-op?

Reply:
`core.Deployment.is_schedule_active` is a bool. 

`client.create_deployment` has `is_schedule_active` as a `bool = None`. The goal here is to have it using using the server default if None is provided. 

Inside of there we do `DeploymentCreate(..., is_schedule_active=is_schedule_active)` which, if we pass `None` without the validator, results in 

```
E   pydantic.error_wrappers.ValidationError: 1 validation error for DeploymentCreate
E   is_schedule_active
E     none is not an allowed value (type=type_error.none.not_allowed)
```

Reply:
@madkinsz I think I addressed your concerns here: https://github.com/PrefectHQ/prefect/pull/7430/files/bc7b7d78d2df68fbfada5d0f93b18d8dff7c4501..7b3b2d4ecb4504cec6f530661b6413fc360ff165#diff-99d27244d265f50d916c514ca3d3afdb591ab196fb390cea38f3b55c2e11847c